garlic/bus/1.0/config/packages/garlic.yaml

```diff
@@ -0,0 +1,13 @@
+enqueue:
```

Comment:
This corresponds to another package than Garlic, you shouldn't post this config here, as there is another one for Enqueue.

Reply:
Thanx! It just mistake. I'll change it

Reply:
@Pierstoval Already changed!